src/NSwag.Commands/Commands/Generation/OpenApiGeneratorCommandBase.cs

```diff
@@ -177,22 +177,22 @@ public string InfoVersion
         [Argument(Name = "DocumentTemplate", IsRequired = false, Description = "Specifies the Swagger document template (may be a path or JSON, default: none).")]
         public string DocumentTemplate { get; set; }
 
-        [Argument(Name = "DocumentProcessors", IsRequired = false, Description = "The document processor type names in the form 'assemblyName:fullTypeName' or 'fullTypeName').")]
+        [Argument(Name = "DocumentProcessors", IsRequired = false, Description = "The document processor type names in the form 'assemblyName:fullTypeName' or 'fullTypeName'.")]
         public string[] DocumentProcessorTypes { get; set; } = new string[0];
 
-        [Argument(Name = "OperationProcessors", IsRequired = false, Description = "The operation processor type names in the form 'assemblyName:fullTypeName' or 'fullTypeName').")]
+        [Argument(Name = "OperationProcessors", IsRequired = false, Description = "The operation processor type names in the form 'assemblyName:fullTypeName' or 'fullTypeName' or ':assemblyName:fullTypeName' or ':fullTypeName'. Begin name with ':' to prepend processors to filter out other operations.")]
```

Comment:
Change last part to “(required when used to filter out other operations)”
There might be scenarios where you want to prepend but not because of filtering (though it is the most common scenario)

Reply:
Description text updated.